astropy/io/fits/tests/test_header.py

```diff
@@ -180,24 +180,24 @@ def test_assign_boolean(self):
         h = fits.Header()
         h.update('FOO', True)
         h.update('BAR', False)
-        assert h['FOO'] == True
-        assert h['BAR'] == False
+        assert h['FOO']
+        assert h['BAR'] is False
```

Comment:
I think here the test change is correct, but it should be `assert h['FOO'] is True`